File: sol/solution/0001-0100/0026/Solution.dart
class Solution {
  int removeDuplicates(List<int> nums) {
    if (nums.isEmpty) return 0;
    int write = 1;
    for (int i = 1; i < nums.length; i++) {
      if (nums[i] != nums[write - 1]) {
        nums[write] = nums[i];
        write++;
      }
    }
    return write;
  }
}

File: sol/solution/0001-0100/0026/Solution.scala
object Solution {
    def removeDuplicates(nums: Array[Int]): Int = {
       var i = 0
       var j = 1
       while(j<nums.length){
        if(nums(i) < nums(j)) {
            nums(i+1) = nums(j)
            i+=1
        }
        j+=1
       }
       i+1
    }
}